app/routers/weekview.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime, timedelta
+from typing import List
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.templating import Jinja2Templates
+from sqlalchemy.orm.session import Session
+
+from app.database.database import get_db
+from app.database.models import User
+from app.dependencies import TEMPLATES_PATH
+from app.routers.dayview import dayview, get_events_and_attributes
+
+
+templates = Jinja2Templates(directory=TEMPLATES_PATH)
+
+
+router = APIRouter()
+
+
+def get_week_dates(firstday: datetime) -> List[datetime]:
```

Comment:
This isn't a route - Maybe it should be in the `internal` dir?

Reply:
whats the diffrent between this func and the other functions in the other routers?